src/parser/warrior/arms/CombatLogParser.js

```diff
@@ -1,21 +1,49 @@
 import CoreCombatLogParser from 'parser/core/CombatLogParser';
 import DamageDone from 'parser/shared/modules/DamageDone';
 
+// Features
+import Checklist from './modules/features/Checklist/Module';
 import Abilities from './modules/Abilities';
 import AlwaysBeCasting from './modules/features/AlwaysBeCasting';
-import BattleCryDamage from './modules/core/BattleCryDamage';
-import BattleCryGCD from './modules/core/BattleCryGCD';
 import CooldownThroughputTracker from './modules/features/CooldownThroughputTracker';
```

Comment:
it doesn't look like arms has any cooldown tracking implemented. this should be cut out for now to avoid an empty cooldowns tab (see image)
![screenshot from 2018-11-04 19-03-03](https://user-images.githubusercontent.com/4909458/47971896-43952780-e064-11e8-9c2f-26519e665645.png)

Reply:
#2589 adds cooldown tracking for debuffs, so once that is merged you could add Colossus Smash and War Breaker.

Reply:
Alright, i'll add that in a futur PR, thank's ! I've put the Cooldowns Tab on comment for now